external/runtime/runtime.depproj

```diff
@@ -82,7 +82,10 @@
 
     <ItemGroup>
       <_CoreCLRSymbolPackagesToDownload Include="@(ReferenceCopyLocalPaths->'%(NuGetPackageId)')" Condition="$([System.String]::Copy('%(Identity)').EndsWith('System.Private.CoreLib.dll'))">
-        <Url>https://dotnet.myget.org/F/dotnet-core/symbols/%(NuGetPackageId)/%(NuGetPackageVersion)/</Url>
+        <!-- If there is an asset root URL set (for orchestrated build), then append symbols to that to grab the coreclr symbol location.  The orchestrated build uses a flat package layout
+             layout. -->
+        <Url Condition="'$(DotNetAssetRootUrl)'!=''">$(AssetRootUrl)symbols/%(NuGetPackageId).%(NuGetPackageVersion).symbols.nupkg</Url>
```

Comment:
Is `AssetRootUrl` defined somewhere or did you mean `DotNetAssetRootUrl`?

Reply:
Was renaming things before pushing.   Fixed.